gossip/poi.go

```diff
@@ -92,8 +92,9 @@ func (s *Service) UpdateStakerPOI(stakerID idx.StakerID, stakerAddress common.Ad
 		s.store.SetStakerPOI(stakerID, common.Big0)
 		return // avoid division by 0
 	}
-	poi := new(big.Int).Mul(weightedFee, lachesis.PercentUnit)
-	poi.Div(poi, s.store.GetPoiFee(poiPeriod))
+	poi := weightedFee // no need to rebase validator's PoI as <= 1.0 ratio
```

Comment:
- add a comment: poi = DFee + VFee * w / W
- check CalcTotalStake() to avoid div by 0.